### DISCUSSION TITLE
Addons

### DISCUSSION THREAD

**Discussion by rottmann** [vitebook/vitebook#56] [Ideas]
It would be nice to gather her information about functionalities and features you need and how we can work together to develop this features.

Everybody has spare time, so we can use the discussion section to talk about and coordinate things.

I created a small (quick & dirty) extension to define controls, action, methods, events, ... (see it here https://i.imgur.com/ClxBM5k.png).
Are there others that need such thing?
Currently it is a really basic addon, and needs much work before public usage is ok. I don't have the time for that, but i would contribute to a project, that develops and maintain such a project.

**Comment by Myrmod** [comment]
I would love to see this happen. Can you share you addon code in the meantime please? :D

**Comment by rottmann** [comment]
@Myrmod OK
[vitebook-addon.tar.gz](https://github.com/vitebook/vitebook/files/8287933/vitebook-addon.tar.gz)
Copy the 2 folders into your `.vitebook` config folder, view the example for config.

**Reply by rottmann** [reply]
@Myrmod little update, i forgot properties.

**Comment by benjazehr** [comment]
It would be great to have a React `prop-types` Addon that shows the PropTypes of a given component.
